### ISSUE TITLE
Request: Add uberjar to release artifacts

### ISSUE THREAD

**Issue by stelcodes** [borkdude/jet#91]
I'm trying to package Jet for NixOS and it's pretty tricky without an uberjar because NixOS doesn't allow the build process to have network access, so libraries cannot be downloaded when building. The prebuilt Linux executable also doesn't work because of missing dynamically linked libraries. Having an uberjar available makes it possible to build a GraalVM binary pretty straightforwardly. That's how [Babashka is packaged](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/interpreters/clojure/babashka.nix#L88) and how I recently [packaged zprint](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/tools/zprint/default.nix#L39).

So my request is for a jet uberjar to be added to the releases. How does that sound? I'm not familiar with this codebase so I understand there may be complications. Cheers!

**Comment by borkdude** [comment]
@stelcodes Yes, that sounds reasonable to do. I think it can be done as part of the JVM step in the CircleCI step, similar to how clj-kondo does it. You could steal the config from clj-kondo .circleci/config.yml and port it to this project, if you want to do the PR yourself.
